packages/manager/modules/pci/src/components/project/flavors-list/flavor-group.class.js

```diff
@@ -13,7 +13,10 @@ export default class FlavorGroup {
     Object.assign(
       this,
       omit(
-        find(flavors, (flavor) => flavor.osType === image && !flavor.isFlex()),
+        find(
+          flavors,
+          (flavor) => image.includes(flavor.osType) && !flavor.isFlex(),
+        ),
```

Comment:
may be we could use native `Array.find` instead of `Lodash.find()`.
```suggestion
          flavors.find((flavor) => image.includes(flavor.osType) && !flavor.isFlex()),
```

Reply:
As discuss together, ok for me